### ISSUE TITLE
accessibility issue using dataAdaptor

### ISSUE THREAD

**Issue by waghanil87** [select2/select2#5776]
I am using select2 V4.0.12, we are loading, I am able to achive multiselect with lazyloading and everything works fie, but problem is when we try to access using screen reader like JWAS and Chrome VOX, it does not read option and looses focus.

Please find the stack overflow link as well
https://stackoverflow.com/questions/60279373/select2-accessibility-issue-using-dataadaptor

I dont find much example on google using data-adaptor methods which implements, multiselect with lazy-loading. Please help with link if you have any

**Comment by kevin-brown** [comment]
It honestly looks like you should actually be using the stock data adapters for Select2 but instead of trying to hack around custom filtering with array data, do it with the AJAX adapter and make your own custom transport. That way you can read from the local array instead of reaching out to a remote data source, and you'll still get pagination (which it looks like you're looking for).

---

This appears to be more of a usage question than a bug report or common feature request.  As we continue to keep the Select2 issue tracker geared towards these two areas, bug reports and common feature requests, we are redirecting usage questions to other communities such as the [mailing list][ml], [IRC][irc], or [Stack Overflow][so].

[ml]: https://select2.org/troubleshooting/getting-help
[irc]: https://select2.org/troubleshooting/getting-help
[so]: http://stackoverflow.com/questions/tagged/jquery-select2?sort=votes
